srcpkgs/gek/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'gek'
+pkgname=gek
+version=0.1
+revision=1
+wrksrc="gek-v${version}"
+build_style=go
+go_import_path="git.sr.ht/~geb/gek"
+go_package="${go_import_path}/src"
+short_desc="Stack-based stream editor and scripting language"
+maintainer="John Gebbie <me@johngebbie.com>"
+license="GPL-3.0-only"
+homepage="https://geklang.org/"
+distfiles="https://git.sr.ht/~geb/gek/archive/v${version}.tar.gz"
+checksum="3a271b067793099a9ad873cd6f9113b3bdc376160e1bdb9f72892e94d0c8f1c6"
+
+# The default would install it called src.
+do_build() {
+	go build -o gek "${go_package}"
+}
```

Comment:
you can easily rename it in do/post_install

Reply:
Ok I've done it. I'm going to bed now, thanks for the heads ups.